### PR TITLE
environment details update

### PR DESCRIPTION
**Pull request by katamaks**

### CODE REVIEW

**Comment by katamaks** [comment]
Ready for review by @jtwaleson

File: content/developerportal/deploy/environments-details.md [hunk 1]
```diff
@@ -5,41 +5,81 @@ description: "This page describes the environmental details of your App."
...
+   * Orange - the environment has warnings
+   * Red - the environment is not working correctly
+
+   ![](attachments/environment-status.png)   
+
+* Running since date - the date that the app has been started
+* Mode - type of environment: production/acceptance/test
+* URL - the URL of the app
+* Custom Domains - the custom domains of the app
+* Java version - until Mendix Desktop Modeler 5.17 the version of java is 1.7. In Mendix Desktop Modeler 5.18 you can choose between java version 1.7 and 1.8. Starting from Mendix Desktop Modeler 6 you cannot choose the java version and it's always version 1.8.
```
Comment (jtwaleson) [inline content/developerportal/deploy/environments-details.md]: Mendix Desktop Modeler -> Mendix version (multiple times)

Comment (jtwaleson) [inline content/developerportal/deploy/environments-details.md]: capitalize Java

File: content/developerportal/deploy/environments-details.md [hunk 2]
```diff
@@ -5,41 +5,81 @@ description: "This page describes the environmental details of your App."
...
+* Runtime version - Mendix Desktop Modeler version
```
Comment (jtwaleson) [inline content/developerportal/deploy/environments-details.md]: Mendix version

File: content/developerportal/deploy/environments-details.md [hunk 3]
```diff
@@ -5,41 +5,81 @@ description: "This page describes the environmental details of your App."
...
+
+## 2.2 Scaling - Mendix Cloud v4
+
+![](attachments/scale.png)
+
+If your app is hosted in Mendix Cloud v4, you will see a section named **Scaling** in the **General** tab.
+
+In **Scaling** there are two sliders that you can control:
+
+* The **Instances** slider to increase or decrease the amount from one to four instances by sliding the slider. The instances are meant for spreading your memory out. You can spread the RAM memory among 4 instances if you have more than 1GB of RAM memory.
```
Comment (jtwaleson) [inline content/developerportal/deploy/environments-details.md]: Add: the minimum amount of memory per instances is 1GB.

Comment (jtwaleson) [inline content/developerportal/deploy/environments-details.md]: are meant for spreading your memory out -> are meant for improved resiliency and increased processing power

File: content/developerportal/deploy/environments-details.md [hunk 4]
```diff
@@ -5,41 +5,81 @@ description: "This page describes the environmental details of your App."
...
+* The **Memory per instance** slider to increase or decrease the memory amount per instance by sliding the slider.
+
+The **Total Allocated Memory** is a process cirkle that shows how much memory is currently used for scaling.
```
Comment (jtwaleson) [inline content/developerportal/deploy/environments-details.md]: circle

File: content/developerportal/deploy/environments-details.md (hunk 1 above)
Comment (katamaks) [inline content/developerportal/deploy/environments-details.md]: done

File: content/developerportal/deploy/environments-details.md (hunk 2 above)
Comment (katamaks) [inline content/developerportal/deploy/environments-details.md]: done

File: content/developerportal/deploy/environments-details.md (hunk 3 above)
Comment (katamaks) [inline content/developerportal/deploy/environments-details.md]: done

Comment (katamaks) [inline content/developerportal/deploy/environments-details.md]: done

File: content/developerportal/deploy/environments-details.md (hunk 4 above)
Comment (katamaks) [inline content/developerportal/deploy/environments-details.md]: stupid me...done

**Comment by katamaks** [comment]
@jtwaleson please approve review

**Comment by katamaks** [comment]
@Adam-Dupaski please merge this after #600 and #604 have been merged

**Pull request merged**